sumologic-kubernetes-setup/v3.0.0/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM registry.access.redhat.com/ubi8/ubi:8.4
+
+### Required Labels #####
+LABEL name="SumoLogic Kubernetes Setup" \
+      vendor="SumoLogic" \
+      version="1.0" \
```

Comment:
We are setting version to the version of used component so here it should be `3.0.0`

Reply:
Done, thanks!